android/src/test/java/com/revenuecat/purchases/common/PeriodTests.kt

```diff
@@ -0,0 +1,24 @@
+package com.revenuecat.purchases.common
+
+import android.os.Build
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [Build.VERSION_CODES.P])
```

Comment:
Android 29 requires JDK 9 to run tests, that's why I specified this version here